### ISSUE TITLE
This package should be deprecated?

### ISSUE THREAD

**Issue by yurivict** [ktosiek/pytest-freezegun#43]
```pytest-freezegun``` systematically causes the ```INTERNALERROR``` failures in PyTest.
See details here: https://github.com/pytest-dev/pytest/issues/10846

**Comment by JoelClemence** [comment]
I believe it just needs #39 to be resolved currently CI is failing, which may be due to #33, probably because it supports really old versions of python.

Looks like this is no longer maintained and so if it remains so a deprecation probably is best course of action

**Comment by jaraco** [comment]
Yes. This package is abandoned. See https://github.com/ktosiek/pytest-freezegun/issues/19#issuecomment-1500919278. So sad to see this name lost to history.
